src/shared-styles/_normalize.scss

```diff
@@ -71,3 +71,9 @@ strong {
 p {
   @include space.block();
 }
+
```

Comment:
Move to before the style rules. Then if a style rule in this file uses it, things won't break.